projects/addon-doc/components/aside/aside.component.html

```diff
@@ -0,0 +1,20 @@
+<ng-content></ng-content>
+<nav
+    tuiTabs
+    vertical="right"
+    [(activeItemIndex)]="activeIndex"
+>
+    <button
+        *ngFor="let item of menuList; let index = index"
+        tuiTab
+    >
+        <a
+            class="t-link"
+            [class._active]="activeIndex === index"
+            [fragment]="item.id"
+            [routerLink]="link | async"
```

Comment:
There must be a current page for routerLink so you do not have to calculate it yourself. Maybe `routerLink="."`?